docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

```diff
@@ -0,0 +1,139 @@
+
+* Perform a clean install of the AksHci PowerShell module. To install the AksHci Powershell module found in the zip file you just downloaded follow [this guide.](https://docs.microsoft.com/en-us/azure-stack/aks-hci/setup-powershell#step-1-download-and-install-the-akshci-powershell-module)
+
+ 
+## Deployment
+
+* Before running the PowerShell Script to deploy your AKS cluster, it is important to verify each physical node of your Azure Stack HCI to validate all of the requirements are satisfied. To do that, open PowerShell as an administrator and run:
+
+  ```powershell
+  Initialize-AksHciNode
+  ```
+
+* Now that all nodes are ready, you will deploy an AKS cluster to your Azure Stack HCI using this [PowerShell script](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_stack_hci/powershell/aks_hci_deploy.ps1). Note, that the script will deploy a simple DHCP based cluster on your Azure Stack HCI, there are additional optional parameters that you could use to customize the deployment to your own environment as described [here] (https://docs.microsoft.com/en-us/azure-stack/aks-hci/setup-powershell). 
+
+![Screenshot showing the AKS on HCI deployment script](./01.png)
+
+* To run the script open PowerShell as an administrator, navigate to the [script folder](https://github.com/microsoft/azure_arc/blob/main/azure_arc_k8s_jumpstart/aks_stack_hci/powershell/) and run: 
+
+  ```powershell
+  .\aks_hci_deploy.ps1
+  ```
+![Screenshot showing the script input](./02.png)
+
+* The script will prompt to provide a set of parameters to configure your cluster: 
+    - **imageDir:** path to the directory where AKS on Azure Stack HCI will store its VHD images, provide a shared path or SMB for multinode
+    - **cloudConfigLocation:** path to the directory where the cloud agent will store its configuration, provide a shared path or SMB for multinode
+    - **vnetName:** the name of the virtual switch to connect the virtual machines to. If you already have an external switch on the host, you should pass the name of the switch here. 
+    - **clusterName:** a name for your AKS cluster, must be lowercase. 
+    - **controlPlaneNodeCount:** number of nodes for your control plane, should be an odd number 1, 3 or 5.
+    - **linuxNodeCount:** number of Linux node VMs for your cluster, if you do not need Linux nodes input 0.
+    - **windowsNodeCount:** number of Windows node VMs for your cluster, if you do not need Windows nodes input 0.
+
+    > **Note: the script may take upto 30 minutes to run**
+
+* Once the script has completed its run, you will see an output as follows: 
+
+![Screenshot showing the script output](./03.png)
+
+* To be able to connect to the Azure Kubernetes Service cluster run the command: 
+
+  ```powershell
+  Get-AksHciCredential -clusterName <cluster_name>
+  ```
+![Screenshot showing the output retrieving credentials](./04.png)
+
```

Comment:
Consider adding more screenshots of the HCI cluster in the Windows Admin Center to provide the reader how an end result should look like

Reply:
I did not use Windows Admin Center and don't even have it installed as it was another procedure, not sure if it looks the same in a real life deployment compared to nested, I'll try